eva/binder/binder_utils.py

```diff
@@ -107,3 +109,60 @@ def check_table_object_is_video(table_ref: TableRef) -> None:
     if not is_video_table(table_ref.table.table_obj):
         err_msg = "GROUP BY only supported for video tables"
         raise BinderError(err_msg)
+
+
+def get_table_ref_subclass(table_ref: TableRef) -> TableRef:
+    if table_ref.is_table_atom():
```

Comment:
I tried but couldn't get it to work.Since we are changing the same node object which is being passed as an argument, it will work only if I assign the child object to something. Which is why I do it in the bind_select_stmnt method.